devservices/inspector/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess you can remove the project. How do you develop and unit test ? Against AWS directly, if so, there is no  dev services story for now.

Reply:
Done. Yes, unfortunately there are no tests except the `InspectorSyncClientFullConfigTest`.